server/__tests__/signupAPI.test.js

```diff
@@ -0,0 +1,98 @@
+
+import { expect } from 'chai';
+import supertest from 'supertest';
+import mongoose from 'mongoose';
+import { Mockgoose } from 'mockgoose';
+import app from '../src/app/app';
+import getMockUser from '../src/utils/testUtils';
+
+const { TEST_PORT = 8888, DB_URL = 'mongodb://localhost:27017/testdb' } = process.env;
```

Comment:
Please import env variables from a common place, check [this](https://stackoverflow.com/a/48828102/10322565) for further info